txscript/standard_test.go

```diff
@@ -1247,7 +1247,7 @@ func TestGenerateSSGenVotes(t *testing.T) {
 	}
 }
 
-// mustExpectedAtomicSwapData is a convenience function the converts that passed
+// mustExpectedAtomicSwapData is a convenience function the converts the passed
```

Comment:
missed the first the => that
"is a convenience function _that_ converts the passed..."